README.md

```diff
@@ -120,9 +120,13 @@ RELEASES
 
 imscJS is released as an NPM package under [imsc](https://www.npmjs.com/package/imsc). The `dev` distribution tag indicates pre-releases.
 
-All builds are available on the [unpkg](https://unpkg.com/) CDN under the `dist` directory.
+All latest builds/dist are available on the [unpkg](https://unpkg.com/) CDN under the [`dist`](https://unpkg.com/browse/imsc/dist/) directory.
 
```

Comment:
I would add a link to the releases on GitHub, which contain links to unpkg and npmjs:

https://github.com/sandflow/imscJS/releases